test/functional/tests/stress/test_stress_io_small_device.py

```diff
@@ -0,0 +1,111 @@
+#
+# Copyright(c) 2019 Intel Corporation
+# SPDX-License-Identifier: BSD-3-Clause-Clear
+#
+
+import pytest
+from datetime import timedelta
+
+from api.cas import casadm, casadm_parser
+from api.cas.cache_config import CacheLineSize, CacheMode, CleaningPolicy
+from core.test_run import TestRun
+from storage_devices.disk import DiskType, DiskTypeLowerThan, DiskTypeSet
+from test_tools.fio.fio import Fio
+from test_tools.fio.fio_param import CpusAllowedPolicy, IoEngine, ReadWrite
+from test_utils.size import Size, Unit
+
+stress_time = timedelta(minutes=30)
+
+
+@pytest.mark.parametrize("cores_number", [1, 4])
+@pytest.mark.parametrize("cache_config", [(CacheMode.WT, None),
+                                          (CacheMode.WA, None),
+                                          (CacheMode.PT, None),
+                                          (CacheMode.WB, CleaningPolicy.acp),
+                                          (CacheMode.WB, CleaningPolicy.alru),
+                                          (CacheMode.WB, CleaningPolicy.nop),
+                                          (CacheMode.WO, CleaningPolicy.acp),
+                                          (CacheMode.WO, CleaningPolicy.alru),
+                                          (CacheMode.WO, CleaningPolicy.nop)])
+@pytest.mark.parametrize("cache_line_size", CacheLineSize)
+@pytest.mark.require_disk("cache", DiskTypeSet([DiskType.optane, DiskType.nand]))
+@pytest.mark.require_disk("core", DiskTypeLowerThan("cache"))
+def test_stress_small_cas_device(cache_line_size, cores_number, cache_config):
+    """
+        title: Stress test for verifying data on small CAS devices.
+        description: |
+          Validate the ability of CAS to handle many iops when device is small
+          using different cache modes, cache line sizes and core numbers.
+        pass_criteria:
+          - No system crash.
+          - Md5 sums of core device and exported object are equal.
+    """
+    cache_mode = cache_config[0]
+    cleaning_policy = cache_config[1]
+
+    with TestRun.step(f"Prepare 1 cache and {cores_number} core devices."):
+        cache_dev = TestRun.disks['cache']
+        cache_dev.create_partitions([Size(100, Unit.MebiByte)])
+        cache_part = cache_dev.partitions[0]
+        core_dev = TestRun.disks['core']
+        core_sizes = []
+        for i in range(cores_number):
+            core_sizes.append(Size(200, Unit.MebiByte))
```

Comment:
```python
core_sizes = [Size(200, Unit.MebiByte)] * cores_number
```
-2 to lines count
+120 to pythonicity

Reply:
One more time You reminded me simple techniques.